<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sewage monitoring</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: #FFFFFF;
}
QToolButton{
    color:green;
    background-color:#FFFFFF;
    border: 1px solid #FFFFFF;
    border-radius: 16px;
	font-size:10px;
}
QToolButton:hover{
    background-color:#F5F5F5;
}

QToolButton:pressed{
    background-color:#555555;
}
QLabel{
	font: 75 16pt &quot;Helvetica&quot;;
    color:green;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,10,0">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(210, 210, 210);
}
QToolButton{
    color:green;
    background-color:#FFFFFF;
    border: 1px solid rgb(225,225,225);
    border-radius: 16px;
}
QToolButton:hover{
    background-color:#F5F5F5;
}

QToolButton:pressed{
    background-color:#555555;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="5,0,0,0,0">
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>340</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="nameLabel">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 75 16pt &quot;Helvetica&quot;;
    color:green;
}</string>
        </property>
        <property name="text">
         <string>EBULENT SEWAGE MONITORING SYSTEM</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>273</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="settingButton">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
    color:green;
    background-color:rgb(220,220,220);
    border: 1px solid rgb(240,240,240);
    border-radius: 16px;
}
QPushButton:hover{
    background-color:#FFFFFF;
}

QPushButton:pressed{
    background-color:#555555;
}</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="bottomWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(220, 220, 220);
}
QToolButton{
    color:black;
    background-color:rgb(220,220,220);
    border: 1px solid rgb(240,240,240);
    border-radius: 16px;
}
QToolButton:hover{
	background-color:#FFFFFF;
}

QToolButton:pressed{
    background-color:#555555;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QToolButton" name="itemToolButton">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Curve</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="itemToolButton_2">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>item 2</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="homeToolButton">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="itemToolButton_3">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>item 3</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="itemToolButton_4">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>item 4</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QAnimationStackedWidget" name="centerStackWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: #FFFFFF;
}

QLabel{
	font: 12pt &quot;Helvetica&quot;;
    color:green;
}</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="centerStackWidgetPage1">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_19" rowstretch="0,10,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="titleWIdget" native="true">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_21">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_22">
            <item row="0" column="4">
             <widget class="QLabel" name="nextTimeLabel">
              <property name="text">
               <string>下次维护日期：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="runTimeLabel">
              <property name="text">
               <string>设备运行时间：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="persionLabel">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>责任人：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="runTimeLineEdit">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="timeLabel">
              <property name="text">
               <string>日期时间：</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7">
             <widget class="QLineEdit" name="persionLineEdit">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="text">
               <string>xxx</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="timeLineEdit">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QLineEdit" name="nextTimeLineEdit">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCustomPlot" name="sewagePlot" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::DashLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>3</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>污染物类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>当前值</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>预警值</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="centerStackWidgetPage2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 0);</string>
      </property>
     </widget>
     <widget class="QWidget" name="centerStackWidgetHome">
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,2,1" columnstretch="1,2,1">
       <item row="1" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="timeWidget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(200, 200, 200);
border: 0px solid #FFFFFF;
border-radius: 16px;</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10" rowstretch="3,2">
          <property name="topMargin">
           <number>32</number>
          </property>
          <property name="bottomMargin">
           <number>32</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_8" columnstretch="0,0">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QLabel" name="hourLabel">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>90</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Helvetica</family>
                    <pointsize>-1</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
    background-color:#A5A5A5;
	font-size:48px;
    color:white;
    border-radius: 6px;
}</string>
                  </property>
                  <property name="inputMethodHints">
                   <set>Qt::ImhDigitsOnly</set>
                  </property>
                  <property name="text">
                   <string>09</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label">
                  <property name="maximumSize">
                   <size>
                    <width>15</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
	font-size:48px;
    color:white;
    border-radius: 0px;
}</string>
                  </property>
                  <property name="text">
                   <string>:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="minuteLabel">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>90</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
    background-color:#A5A5A5;
	font-size:48px;
    color:white;
    border-radius: 6px;
}</string>
                  </property>
                  <property name="inputMethodHints">
                   <set>Qt::ImhDigitsOnly</set>
                  </property>
                  <property name="text">
                   <string>00</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_7" rowstretch="3,1">
                <item row="1" column="0">
                 <widget class="QLabel" name="secondLabel">
                  <property name="minimumSize">
                   <size>
                    <width>30</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
    color:white;
    font-size:32px;
    border-radius: 8px;
}</string>
                  </property>
                  <property name="text">
                   <string>12</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>36</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_5" columnstretch="5">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="dateLabel">
              <property name="font">
               <font>
                <family>Helvetica</family>
                <pointsize>-1</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	font-size:36px;
    color:white;
    border-radius: 8px;
}</string>
              </property>
              <property name="text">
               <string>2017 年 07 月 20 日 星期 四</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="centerStackWidgetPage4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
     </widget>
     <widget class="QWidget" name="centerStackWidgetPage5">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 127);</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QAnimationStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header location="global">qanimationstackedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>persionLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
